styles/alert.css

```diff
@@ -0,0 +1,29 @@
+:root {
+  --primary: #acc8e5;
+  --secondary: #e06358;
+}
+
+.alert {
+  padding: 1rem 0.9rem;
```

Comment:
Use global variables for spacing for properties like margin, padding, and gap to maintain consistency across the website